tests/Component/Basket/BasketBuilderTest.php

```diff
@@ -57,25 +58,25 @@ public function testBuildWithNonExistentProductCode()
 
         $paymentPool = new PaymentPool();
 
-        $addressManager = $this->getMock('Sonata\Component\Customer\AddressManagerInterface');
+        $addressManager = $this->createMock('Sonata\Component\Customer\AddressManagerInterface');
 
         $basketBuilder = new BasketBuilder($productPool, $addressManager, $deliveryPool, $paymentPool);
 
-        $basketElement_1 = $this->getMock('Sonata\Component\Basket\BasketElementInterface');
+        $basketElement_1 = $this->createMock('Sonata\Component\Basket\BasketElementInterface');
```

Comment:
can you fix this variable name to `$BasketElement1` as we don't use underscores for variable names, thank you